src/logic/editable-storage-logic.ts

```diff
@@ -22,4 +22,18 @@ export class EditableStorageLogic {
       await this.vmManager.putContractStorage(toFancyAddress(this.contractAddress), fromHexString(slot.key), fromHexString(slot.val));
     }
   }
+
+  async setVariables(variables: any = {}) {
+    if (variables === undefined || variables === null) return;
+
+    let slots: any[] = [];
+
+    for (const [variableName, variableValue] of Object.entries(variables)) {
+      slots = slots.concat(computeStorageSlots(this.storageLayout, { [variableName]: variableValue }));
```

Comment:
Alternatively maybe we could do `await setVariable(variableName, variableValue)`.

Reply:
Thanks! I pushed the fix.